_posts/2024-01-10-practical_zephyr_basics.md

```diff
@@ -210,27 +210,27 @@ Instead of setting `ZEPHYR_BASE` manually as, e.g., done by the `env.sh` script
 
 The final command, `west zephyr-export`, is a ["Zephyr extension command"](https://docs.zephyrproject.org/latest/develop/west/zephyr-cmds.html#installing-cmake-packages-west-zephyr-export) that installs the required _CMake_ packages that we'll need in the next steps when using `find_package` to locate Zephyr's sources in a `CMakeLists.txt` file.
 
-That's it for the environment, we can now go ahead and create applications. One last time I'd like to emphasize that this was just a warm-up to get started with Zephyr. It is not required or recommended to use such a `setup.sh` script, it's just one of many possibilities you have when bringing up your development environment.
+That's it for the environment; we can now create applications. One last time, this was just a warm-up to start with Zephyr. It is not required or recommended to use such a `setup.sh` script; it's just one of many possibilities you have when bringing up your development environment.
 
 
 
 ## Creating an empty application skeleton
 
-Now that we have a working installation, we can start with the important parts of this article series. Before creating our first files, let's review Zephyr's supported application _types_.
+Now that we have a working installation, we can start with the important parts of this article series. Before creating our first files, review Zephyr's supported application _types_.
```

Comment:
Here I'd maybe keep the "let's" or phrase it differently since otherwise it sounds like an instruction for the reader.

Reply:
Good call, definitely a better read this way.